packages/knip/src/plugins/biome/index.ts

```diff
@@ -0,0 +1,49 @@
+import type { IsPluginEnabled, Plugin, PluginOptions, ResolveConfig } from '../../types/config.js';
+import { toDeferResolve, toDependency, toIgnore, type ConfigInput, type Input } from '../../util/input.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { BiomeConfig } from './types.js';
+
+const title = 'biome';
+
+const enablers = ['@biomejs/biome'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => 
+  {
+    const enabled = hasDependency(dependencies, enablers);
+    return enabled;
+  }
+
+const config: string[] = ['biome.json'];
+
+
+const resolveExtends = (extendsArray: string[], options: PluginOptions): Input[] => {
+  const inputs = [] as Input[];
+  for (let item of extendsArray) { 
+    // https://biomejs.dev/guides/configure-biome/#share-a-configuration-file
+    if (item.endsWith(".json")) {
+      inputs.push(toDeferResolve(item));
+    } else {
+      if (require.resolve(item, { paths: [options.cwd] })) {
+        inputs.push(toDependency(item));
+      }
+    }
+  }
+  return inputs
+}
+
+
+const resolveConfig: ResolveConfig<BiomeConfig> = (config, options) => {
+  return [
+    // If we're using biome plugin, biome should be ignored in the report
+    toIgnore("@biomejs/biome", "dependencies"), 
```

Comment:
Isn't the `biome` executable expected in e.g. some `package.json#scripts`? That would add a reference to the `bin` of the `@biomejs/biome` package. For instance, Knip itself has this too: https://github.com/thoroc/knip/blob/22739cee31e0845d8e73aeb26c77a0e75a7e1e71/package.json#L23-L25

Reply:
yeah that's true. let me fix this